### ISSUE TITLE
bad data related to Mar 2-3, Nov 25ish 1980 outliers for MSU_NO6 (channel 4)

### ISSUE THREAD

**Issue by amschne** [NOAA-PSL/build_gsinfo#9]

**Comment by jswhit2** [comment]
@amschne @jack-woollen do we need to simply turn off MSU channel 4 for Mar 2-3  and Nov 25 1980?

**Comment by jack-woollen** [comment]
@jswhit2 @amschne  My recollection is all the TOVS data on those dates is bad and needs to be removed or replaced. I'm off for the weekend but will focus to resolve this in a couple days.

**Comment by jack-woollen** [comment]
@jswhit2 I setup a run on hercules to test some alternate TOVS files found for this period, but there's no sst boundary condition for March 1980 in /work2/noaa/gsienkf/whitaker/era5sstice. 

I saw some reference to sst in 198003 scout run output being on gaea: 
FNTSFA   = "/gpfs/f6/drsa-precip4/world-shared/Jeffrey.S.Whitaker/era5sstice/era5_sst_198003.grib",
but for some reason I can't cd to or even ls  /gpfs/f6 on gaea!  

Probably it needs other files for 1980 too. Are there boundary conditions for March 1980 on hercules or hera somewhere?

**Comment by jswhit2** [comment]
Jack: The files are on HPSS
at /ESRL/BMC/gsienkf/2year/whitaker/era5/sstice. I'll try to get them
copied over to hercules for you this morning.  -Jeff
--------------------------------------------------=---
Jeffrey S. Whitaker         Phone: (307)352-9213
Meteorologist               NOAA Physical Sciences Lab
<http://tinyurl.com/5telg>

**Comment by jswhit2** [comment]
Jack: I'm copying the files now.   BTW - you can request access to the
ira-da project on gaea c6 via aim.rdhpcs.noaa.gov.

-Jeff
--------------------------------------------------=---
Jeffrey S. Whitaker         Phone: (307)352-9213
Meteorologist               NOAA Physical Sciences Lab
<http://tinyurl.com/5telg>

**Comment by jack-woollen** [comment]
@jswhit2 Thanks! I found unhtar.sh in /gpfs/f5/nggps_psd/proj-shared/Jeffrey.S.Whitaker/era5sstice, which I can probably also use to get sst/ice files, when hpss is up. And thanks too, I'll request access to ira-da.

**Comment by jack-woollen** [comment]
@jswhit2 @amschne Ok, I reran the beginning of March with the existing data and it reproduced the problem. I reran with another copy of the data and it had a similar problem. I just ordered fresh copies of the 1B data from NESDIS CLASS and should be getting that delivered today or tomorrow. I need to dig out some conversion codes from HPSS to go from the 1B NESDIS orbits to the 1C radiance format, which hopefully can do later today. Then convert and try rerunning with the fresh data.

**Bottom line:** I received fresh March1980 TOVS orbit data from NOAA CLASS download and converted to bufr. After re-running with the new data, the result is identical to the scout run. I'll make some plots of the bad data to see what it looks like compared to data from the rest of March. It'd be interesting to see what MERRA looks like in this regard.

**Comment by amschne** [comment]
I discussed this with @jack-woollen - recommended to remove these data from NNJA (https://github.com/NOAA-PSL/mountain-flurries/issues/42)
